src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 200) {
```

Comment:
The constructor of the `Animal` class should not set a default health value of `200`. According to the task description, the default health should be `100`. Please adjust the default parameter value to `100`.